src/query/_pages/plan-page-query.gql

```diff
@@ -10,11 +10,27 @@ query PlanPage {
       docId
     }
   }
-  
+
   offers: queryPriceOffer {
     id: docId
     name: details_tag_s
     periodCount: details_periodCount_i
     discountPerc: details_discountPercX10000_i
   }
+
+  getDao(docId: $daoId) {
+    planmanager {
```

Comment:
@pugakn I think this is duplicate as planmanger is already fetched in `dao-active.gql` when app loads. You can move this part there and then fetch it via getter in dao store ?

Reply:
I totally agree! let's see how we can refactor redux/apollo over the next year to clean up the codebase